README.md

```diff
@@ -1,17 +1,20 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://avatars0.githubusercontent.com/u/993323" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
+    </a>
+    <a href="https://github.com/" target="_blank">
+        <img src="https://github.githubassets.com/assets/GitHub-Mark-ea2971cee799.png" height="80px" alt="GitHub">
```

Comment:
![image](https://github.com/yiisoft/actions/assets/525501/43d067f2-b258-4544-b852-9d5923341459)

It looks strange. Need image that looks good both for dark and light themes.

Reply:
Ok